tests/storage/restricted_namespace_cloning/conftest.py

```diff
@@ -252,3 +251,13 @@ def vm_for_restricted_namespace_cloning_test(
     ) as vm:
         running_vm(vm=vm, wait_for_interfaces=False)
         yield vm
+
+
+@pytest.fixture()
+def user_has_get_permissions_in_source_namespace(namespace, unprivileged_client):
+    assert list(DataVolume.get(namespace=namespace.name, dyn_client=unprivileged_client))[0]
```

Comment:
This is not a good idea. You should look at DV by name. You already know which dv was created using unprivileged_client.

Reply:
I only need to verify that unprivileged_client can do `get` calls on DVs in that namespace, but I can add a name.